org.jellyfin.JellyfinServer.yml

```diff
@@ -349,15 +349,13 @@ modules:
           - /lib/pkgconfig
         sources:
           - type: archive
-            url: https://code.videolan.org/videolan/libplacebo/-/archive/v5.264.1/libplacebo-v5.264.1.tar.gz
-            sha256: 9eb0c198f58d994f1c0d173dd37647d4d07d27972412c48ab758e984503c787b
+            url: https://code.videolan.org/videolan/libplacebo/-/archive/v6.292.1/libplacebo-v6.292.1.tar.gz
```

Comment:
@istori1 Please pin the `libplacebo` to v5 until we upgraded to FFmpeg 6.1+ since `libplacebo` v6 API requires Vulkan 1.3, which is not supported in FFmpeg 5.1.3 and 6.0 releases. It can be built but doesn't work.